dev/Installer/src/help.cpp

```diff
@@ -0,0 +1,18 @@
+#include "pch.h"
+#include "help.h"
+
+void show_help()
+{
+    std::wcout << L"Installs the Microsoft Project Reunion framework." << std::endl;
+
+    // Get the program name for the module, since argv[0] is unreliable if launched from CreateProcess.
+    wchar_t programName[MAX_PATH];
```

Comment:
No version of Windows ships with longPathAware enabled at this time. So including this fusion manifest flag just means you now have to support users that manually flip this switch. Noble but you have to decide if it's something you _really want to support_ at this stage.

Reply:
There's the world that is, and the world we desire it to be. We must live in the one as we strive to build the other.

I see no downside to support longPathAware here for the installer.

Windows doesn't enable longPathAware by default because it's not longPathAware-clean yet. It would be unwise to enable it by default until that's addressed. But Project Reunion's installer is a smaller problem space than Windows ;-) so much more tractable to enable.

Reply:
Fixed to use the wil helper.